packages/packages/commonwell-cert-runner/src/patient-management.ts

```diff
@@ -40,29 +42,48 @@ export async function patientManagement(
   console.log(`>>> D4a: Merge two Patient records`);
   // Create a second patient
   const respPatient2 = await commonWell.registerPatient(queryMeta, mergePatient);
-  const patientId2 = getIdPatient(respPatient2);
+  const patientId2 = getIdTrailingSlash(respPatient2);
   const referenceLink = respD1b._links.self.href;
 
   await commonWell.mergePatients(queryMeta, patientId2, referenceLink);
   console.log("D4a successful");
 
+  // LINK MANAGEMENT
```

Comment:
This part of the link management made sense to be included in here because it shared a lot of the logic